dev-requirements.txt

```
@@ -1,10 +1,11 @@
-black==22.10.0
+black==22.12.0
 isort; python_version < "3.8"
 isort==5.12.0; python_version >= "3.8"
-pre-commit==2.20.0
-flake8==5.0.4
+pre-commit==3.0.2
+flake8; python_version < 3.8'
+flake8==6.0.0; python_version >= '3.8'
```

@terriko, sorry, I've missed an opening quote on the first line in my comment 😫
It should actually be:
```suggestion
flake8; python_version < "3.8"
flake8==6.0.0; python_version >= "3.8"
```
Double quotes to be consistent in the file.